src/test/java/uk/gov/pay/api/json/CreateCardPaymentRequestDeserializerTest.java

```diff
@@ -145,6 +145,46 @@ public void deserialize_shouldDeserializeARequestWithDelayedCaptureEqualsFalseSu
         assertThat(paymentRequest.getDelayedCapture(), is(Optional.of(Boolean.FALSE)));
     }
 
+    @Test
+    public void deserialize_shouldDeserializeARequestWithMotoEqualsTrueSuccessfully() throws Exception {
```

Comment:
`CreatePaymentIT.createMOTOPayment` would have passed without these tests. In the future, these tests could go into another PR.